src/CommandHandler.py

```diff
@@ -56,7 +56,8 @@ def ProcessCommand(self, cmdString: str):
         try:
             handler(cmd)
         except Exception as e:
-            print('Handler error - ', e)
+            print('Handler error')
```

Comment:
Any reason in particular for removing this line?

Reply:
Ah, sorry - I was printing out a stack trace instead and forgot to change it back.